trappy/systrace.py

```diff
@@ -75,3 +85,24 @@ def trace_hasnt_finished(self):
 
         """
         return lambda x: not x.endswith("</script>\n")
+
+    def format_data(self, unique_word, data_str):
+        if unique_word != 'tracing_mark_write:':
+            return data_str
+
+        # Disacrd not useful clock synchronization events
+        if 'trace_event_clock_sync' in data_str:
+            return ''
+
+        match = SYSTRACE_EVENT.match(data_str)
+        if match:
+            data_str = "event={} pid={} func={} data={}".format(
+                match.group('event'), match.group('pid'),
+                match.group('func'), match.group('data'))
+            return data_str
+
+        raise ValueError('Unexpected systrace marker: {}'.format(data_str))
```

Comment:
I don't think it should be an error, anyone could put stuff in `tracing_mark_write`. Maybe log a warning?